pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/BaseSingleInputAggregationFunction.java

```diff
@@ -40,7 +40,7 @@ public BaseSingleInputAggregationFunction(ExpressionContext expression) {
 
   @Override
   public String getResultColumnName() {
-    return getType().getName().toLowerCase() + "(" + _expression + ")";
+    return getType().getName().replace("_", "").toLowerCase() + "(" + _expression + ")";
```

Comment:
will this be backward incompatible when upgrading 
(e.g. server returns a different result ColunmName while broker is still using older version of the name and thus can't find the right column?)

Reply:
none of the v1 aggregation function have an '_' in their name so this should be fine. This was added to handle backward incompatibility for BOOL_AND and BOOL_OR which are named with underscore in v2 but don't have an underscore in v1.

Reply:
discussed offline - now we register both names without and without the underscore. removed this change